AppCenterAnalytics/AppCenterAnalytics/TransmissionTarget/MSPropertyConfigurator.h

```diff
@@ -25,6 +25,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)setAppLocale:(nullable NSString *)appLocale;
 
+/**
+ * Set the user identifier.
+ * The user identifier needs to start with c: or i: or d: or w: prefixes.
```

Comment:
Try to fit everything in one line as much as possible otherwise the doc may get truncated by Xcode. To try it out keep the "option/alt" key pressed while clicking the method, Xcode will compute  the documentation.

Reply:
<img width="484" alt="screen shot 2018-11-20 at 1 30 33 pm" src="https://user-images.githubusercontent.com/623672/48804090-8abc2300-ecc8-11e8-815f-4025e01119e0.png">

To me it looks good with the current, not sure what you are asking?

Reply:
Let me try again, not sure which version of Xcode didn't work.

Reply:
Nevermind, that was Xcode 8 picking up the doc from the property instead of the setter :man_facepalming:. Xcode 10.1 takes the right doc. 
<img width="462" alt="screen shot 2018-11-20 at 13 36 34" src="https://user-images.githubusercontent.com/5318867/48804508-a96ee980-ecc9-11e8-81c2-3b27eb727154.png">